hack/istio/install-music-store-demo.sh

```diff
@@ -12,6 +12,8 @@
 : ${DELETE_DEMOS:=false}
 : ${MSTORE:=music-store}
 : ${BASE_URL:=https://raw.githubusercontent.com/kiali/demos/master}
+: ${MINIKUBE_PROFILE=minikube}
+: ${ISTIO_NAMESPACE=istio-system}
```

Comment:
The semantics for `=` is different than `:=` (in fact, I don't know what the rule is if using just `=` here).
So I think you need a `:` character in here - bash has very different semantics depending on how the value is declared here - read this for the gory details: https://tiswww.case.edu/php/chet/bash/bashref.html#Shell-Parameter-Expansion

So you want this:

```suggestion
: ${MINIKUBE_PROFILE:=minikube}
: ${ISTIO_NAMESPACE:=istio-system}
```

Reply:
Thanks, @jmazzitelli !